src/components/shared/TreeView.js

```diff
@@ -141,7 +142,6 @@ class TreeViewRowFixedColumns<DisplayData: Object> extends React.PureComponent<
         {columns.map(col => {
           const RenderComponent = col.component;
           const text = displayData[col.propName] || '';
-
           return (
             <span
```

Comment:
Do you think you can revert the 2 line changes in this file? Thanks

Reply:
Sure

Reply:
That's still here ;)

Reply:
don't worry about that, i'll do the change and land, thanks a lot :)

Reply:
Thanks for being very patient and helpful @julienw